docs/semgrep-appsec-platform/jira.md

```diff
@@ -94,56 +93,68 @@ If you opt for a drop-down or a checkbox issue type field, verify that:
 :::note
-Creating tickets for many findings at once may take some time. Tickets that take longer than 10 seconds to create are shown in the UI once you refresh the page.
+Creating tickets for many findings at once may take some time. Tickets that take longer than 10 seconds to create are shown in Semgrep once you refresh the page.
 :::
 
-![Create Jira ticket - Code](/img/jira-code-findings.png)
+![Create Jira ticket - Code](/img/jira-code-findings.png#md-width)
 ***Figure.*** Code triage flow
 
-Once a ticket has been created, a link appears on the right side of the **Findings** page and along the top of an individual finding's details page.
+Once a ticket has been created, a link appears on the **Findings** page and along the top of an individual finding's details page.
 
-![Jira ticket created - Code](/img/jira-code-ticketed.png)
+![Jira ticket created - Code](/img/jira-code-ticketed.png#md-width)
 ***Figure.*** Code ticket created
 
 ### Supply Chain
 
-:::info
-Only users who’ve enabled Jira previously can create Supply Chain tickets. Semgrep is working on releasing an update that allows all users to create Supply Chain tickets.
-:::
+You can create tickets for Supply Chain findings using the **Triage** button on the:
 
-The Supply Chain triage flow creates tickets for all usages of a given vulnerability. Usages cannot be selected individually.
+* [**Supply Chain > Vulnerabilities**](https://semgrep.dev/orgs/-/supply-chain/vulnerabilities) page
+* Individual finding's **Details** page
 
-To create tickets for one or more Supply Chain findings, you can use the triage button on the:
+To create tickets:
 
-* **Supply Chain > Vulnerabilities** page
-* Individual finding's **Details** page.
+1. If you're on the [**Supply Chain > Vulnerabilities**](https://semgrep.dev/orgs/-/supply-chain/vulnerabilities) page, select the findings for which you want tickets created. Otherwise, proceed to step two.
+2. Click **Triage**.
+3. Set the status to **Fixing**, and select the **Create <span className="placeholder">NUMBER_OF_TICKETS</span> JIRA tickets in <span className="placeholder">PROJECT_NAME</span>** box that appears.
+4. Click **Change status** to proceed.
 
-![Create Jira ticket - Supply Chain](/img/jira-ssc-findings.png)
+![Create Jira ticket - Supply Chain](/img/jira-ssc-findings.png#md-width)
 ***Figure*.** Supply Chain triage flow
 
-![Jira ticket created - Supply Chain](/img/jira-ssc-ticketed.png)
+![Jira ticket created - Supply Chain](/img/jira-ssc-ticketed.png#md-width)
 ***Figure*.** Supply Chain ticket created
 
-:::caution Limitations
-Tickets can only be created for Supply Chain findings with **Reachable** code usages. They cannot be created for **Always reachable**, **Reachable if** (requires manual review), or **Unreachable** findings.
-:::
-
 ### Secrets
 
-After selecting one or more findings, you can use the triage button on the **Secrets** page to create tickets for them.
+To create tickets for findings generated by Semgrep Secrets:
```

Comment:
The previous sentences with this pattern just said "To create tickets". Just an observation.